### ISSUE TITLE
How to specify two modifiers?

### ISSUE THREAD

**Issue by realityking** [lodash/grunt-lodash#20]
This is probably a user error, but I can't seem to be able to add two modifiers, in my case `modern` and `strict`.

Here's the config I'm using:

```
lodash: {
    build: {
        dest: '.tmp/scripts/lodash.custom.js',
        options: {
            modifier: 'modern',
            exports: 'global',
            include: 'intersection',
            flags: ['development']
        }
    }
}
```

Thank you for your help.

**Comment by jdalton** [comment]
@phated It looks like currently you can only provide one modifier? If that's the case we should expand it to accepting an array.

**Comment by realityking** [comment]
Awesome, thank you. Could you tag this?

**Comment by jdalton** [comment]
I'll tag and release this weekend along with `docdown`.

**Comment by realityking** [comment]
@jdalton Any news on that tag? Not urgent, just checking in.

**Comment by jdalton** [comment]
Bumped / published 0.5.0.
